test/lib/BedrockTester.cpp

```diff
@@ -204,6 +204,11 @@ string BedrockTester::executeWaitVerifyContent(SData request, const string& expe
     return results[0].content;
 }
 
+STable BedrockTester::executeWaitVerifyContentTable(SData request, const string& expectedResult) {
```

Comment:
This is a shitty name but I couldn't think of anything better.  Suggestions are welcome

Reply:
These names are all bad, because someone picked "executeWait" a long time ago. They should probably be called "makeRequest" and "makeRequests" or something, but I don't think we need to change them now.